deploy/prometheus/prometheus-alert-rules-clickhouse.yaml

```diff
@@ -22,7 +22,7 @@ spec:
               Please check instance status
               ```kubectl logs -n {{ $labels.namespace }} {{ $labels.pod_name }} -c metrics-exporter -f```
 
-        - alert: ClickHouseServerDown
+        - alert: ClickHouseServerPossibleDown
           expr: chi_clickhouse_metric_fetch_errors > 0
```

Comment:
Can we have the critical alert only system.metrics fetch error? 
Others may happen at running ClickHouse server, and I would like to see an alert but not a down one.

Reply:
yep, I can split this alert for two
critical for fetch_type='system_metrics' and high for all other types